cactusbot/handler.py

```diff
@@ -13,6 +13,10 @@ def __init__(self, *handlers):
 
         self.handlers = handlers
 
+    async def register_handler(self, handler):
```

Comment:
This won't work; `handlers` is a `tuple`, and you're trying to `+=` something to it. (Test your code! >.>)

What might be better is to just make `self.handlers = list(handlers)` in `__init__`, and then just use `handlers.handlers.append(new_handler_thing)`.